paas/metric-exporter/manifest.yml

```diff
@@ -0,0 +1,23 @@
+---
+applications:
+- name: metric-exporter
+  memory: 100M
+  instances: 1
```

Comment:
I think we're running 2 instances of carbon-relay. We haven't done any performance testing and we're leaving the fine tuning for later but the idea was to have multiple just to see how it behaves. Do you think we'll end up with more than on metric-exporter? If so, maybe add another instance?

Reply:
I'm almost certain it will double report if we run more than one instance. It has no state or shared buffer so both instances will be working independently and pulling the same metrics for a given time frame. Carbon relay is fine as it's essentially a forwarding proxy balanced via DNS and we're running multiple instances for HA and so that it can keep up with the metric volume from multiple app instances.

Reply:
yeah that makes sense, good point 👍